app/src/main/java/com/bielczy/mvvmclean/core/app/di/NetworkModule.kt

```diff
@@ -0,0 +1,37 @@
+package com.bielczy.mvvmclean.core.app.di
+
+import com.bielczy.mvvmclean.BuildConfig
+import com.bielczy.mvvmclean.features.data.RickAndMortyApi
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+val networkModule = module {
+
+    single<Interceptor> {
+        HttpLoggingInterceptor()
+            .apply {
+                if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BODY
```

Comment:
W tym miejscu powinno być przypisanie rezultatu z bloku if else do zmiennej `level`. Niestety ten mały błąd wkradł się w wideo, natomiast pod nagraniem znajdziesz wyjaśnienie jak to ostatecznie powinno wyglądać :)